CHANGELOG.md

```diff
@@ -63,19 +63,145 @@ Ansible Changes By Release
 ### New Modules
 
 #### Cloud
+- amazon
+  * aws_acm_facts
+  * aws_kms_facts
+  * aws_s3_cors
+  * aws_ssm_parameter_store
+  * ec2_ami_facts
+  * ec2_elb
```

Comment:
This actually isn't new, there was some renaming confusion early in 2.4, but don't put ec2_elb in the changelog please.